### ISSUE TITLE
Bump version in monorepo with several commits in different workspaces

### ISSUE THREAD

**Issue by julocatelli** [orhun/git-cliff#648]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the bug

I found a bug when bumping the version for a specific workspace.

I have a monorepo with a frontend and a backend projects and both have their own version and their own changelog.

I committed some changes on the frontend and different changes on backend.
When I try to `bump` the version of the backend I get the version of the frontend.

It seems take always the version of the first commit.

The rest of the changelog generation works perfectly by the way :)

### Steps To Reproduce

commit a change in one project `e.g. apps/front`
commit another change in another project `e.g. apps/back`

then try to run
`git cliff versio-prev.. --bump --include-path "apps/back/**/*" --repository "./"`

The output version is the `apps/front` project version bump.

### Expected behavior

it should return the bump version of the target project

### Software information

- Operating system: macos
- Cargo version: 1.76.0

**Comment by holmesb** [comment]
`--tag-pattern` ?

**Comment by orhun** [comment]
Yup, you can use `--tag-pattern` to filter for different versions while bumping.
